src/nl/hannahsten/texifyidea/run/latex/ui/LatexSettingsEditor.kt

```diff
@@ -511,6 +515,10 @@ class LatexSettingsEditor(private var project: Project?) : SettingsEditor<LatexR
         pdfViewer = LabeledComponent.create(viewerField, "PDF viewer")
         panel.add(pdfViewer)
 
+        requireFocus = JBCheckBox("Require focus after compilation")
```

Comment:
We should make it clear what will receive focus: the IDE or the PDF viewer, maybe "Focus PDF viewer after compilation" would be better

Reply:
Although, that's probably not what it's doing at the moment, if we leave it visible for all pdf viewers. "Allow PDF viewer to focus after compilation" would be closer to the truth